### PR TITLE
resolve temperrature conversion

### PR DESCRIPTION
**Pull request by gabrielgiroe1**

### CODE REVIEW

File: 01_temperature/temperature.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    else
+       5.0*(param-32)/9
+  end
+end
+
+def ctof (param)
+  if param===0
+    32
+    else 9.0*param/5.0+32
+  end
```
Comment (pirvudoru) [inline 01_temperature/temperature.rb]: Rewrite this as a one liner.

File: 01_temperature/temperature.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+def ftoc(param)
+  if param ===32
+    0
+    else
+       5.0*(param-32)/9
+  end
```
Comment (pirvudoru) [inline 01_temperature/temperature.rb]: Rewrite this as a one liner.

File: 01_temperature/temperature.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+end
+
+def ctof (param)
+  if param===0
+    32
+    else 9.0*param/5.0+32
```
Comment (pirvudoru) [inline 01_temperature/temperature.rb]: Do you really need the if ? What happens if you delete it and leave the function do only the else branch? (also reformat the code :) )

**Review by pirvudoru — CHANGES_REQUESTED** [review]

**Comment by alex-damian-negru** [comment]
Tip: Use `Ctrl + Alt + L` to have the IDE auto-format the code for you to improve readability.

**Pull request merged**